### DISCUSSION TITLE
2D Game Examples

### DISCUSSION THREAD

**Discussion by aliessmael** [dotnet/maui#8120] [Ideas]
Hi when we can expect to have complete 2d game examples?
I see we can use GraphicsView for 2d drawing which is nice, but not sure what we can use to get input, math, ..., etc.
It will be great to have full 2d game examples.  when we can expect to have that?.

**Comment by GeraudFabien** [comment]
https://dotnet.microsoft.com/en-us/apps/games/engines

**Reply by aliessmael** [reply]
Thanks, but not looking for game engines , I am looking for very simple examples, those examples will help me if I want to make very simple game, custome chart, or custom ui, 2d game examples will be so handy for that.

**Reply by manishbisen** [reply]
Interesting question.
